### PR TITLE
contributing: add reference to google docs style guide

### PR DESCRIPTION
**Pull request by wojcik-dorota**
Adding a reference to the Google developer documentation style guide for the contributors to devportal - style guidelines to be followed if our Aiven-specific rules dont cover a specific stylistic problem.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -69,6 +69,10 @@ Style guide
 
 These are the guidelines that we use for Aiven Docs; following them helps us keep all the content consistent and easy to follow for our audience. We review your pull request against these guidelines, so for fewer rounds of revisions, follow this advice!
 
+.. note::
```
Comment (staceysalamon-aiven) [inline CONTRIBUTING.rst:72]: Would it be better to instead put this in the introduction? My thought is that the introduction would be revised to explain that the Google developer documentation style guide is the default and that exceptions/gaps are detailed in the rest of the style guide.

**Review by staceysalamon-aiven — COMMENTED** [review]
Thank you for doing this! I left a comment with a suggestion that we incorporate this into the introduction.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (wojcik-dorota) [reply CONTRIBUTING.rst:72]: Thanks @staceysalamon-aiven :-) It's an option definitely. Still, if you mean the into to the CONTRIBUTING file, this doc covers a wide range of areas, style guide being only one of them. And an introduction seems to stay a general part for providing info regarding the whole thing, not addressing a specific section. From my pov, style guide info belongs to the style guide chapter, which is actually there, so why not use it. @harshini-rangaswamy , any thought on that?

Comment (staceysalamon-aiven) [reply CONTRIBUTING.rst:72]: Yeah, so I meant the introduction to the style guide, not the whole contributing file. 

image

**Review by harshini-rangaswamy — APPROVED** [review]
Looks good to me with the info about the style guide as a part of the introduction.

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM

**Pull request merged**